packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -112,24 +119,35 @@ class SendConfirmation extends React.Component<Props, State> {
 
   sendOrInvite = (inviteMethod?: InviteBy) => {
     const { amount, reason, recipient, recipientAddress } = this.getConfirmationInput()
-    const { onConfirm } = this.getNavParams()
 
     this.props.sendPaymentOrInvite(
       amount,
       reason,
       recipient,
       recipientAddress,
       inviteMethod,
-      onConfirm
+      () => {
```

Comment:
Can we add this logic to the saga instead of a callback here. We should try to keep our component classes as free of business logic as possible